util/arbmath/math_test.go

```diff
@@ -230,6 +230,29 @@ func TestSaturatingNeg(t *testing.T) {
 	}
 }
 
+func TestApproxExpBasisPoints(t *testing.T) {
```

Comment:
my one comment is maybe first do this test in a PR by itself, then we can merge the ApproxBasisPoints and see it doesn't change any value.

Reply:
I don't want to do this for 3 reasons:

1. I already added the test locally before changing the code under test exactly to ensure that I didn't change the behavior, and it didn't.

2. It seems obvious from inspection that this is almost a documentation-only change. The rest of the diff boils down to:
  a. A rename of "bips" to "b"
  b. Changing the loop to count down from accuracy instead of up from 1.
  
3. Extra PRs means extra time waiting on PR review. 😢